apps/global-find-replace/src/components/SearchForm.tsx

```diff
@@ -0,0 +1,142 @@
+            isChecked={filters.caseSensitive}
+            onChange={(e) => onFiltersChange({ caseSensitive: e.target.checked })}
+            className={css({ marginTop: '12px' })}>
+            Case sensitive
+          </Checkbox>
+        </Flex>
+
+        <Flex flexDirection="column">
+          <FormLabel htmlFor="replace">Replace</FormLabel>
+          <TextInput id="replace" value={filters.replace} onChange={(e) => onFiltersChange({ replace: e.target.value })} onKeyDown={handleKeyDown} />
+          <Flex flexDirection="row">
+            <Checkbox
+              id="includeAllFields"
+              isChecked={filters.includeAllFields}
+              onChange={(e) => onFiltersChange({ includeAllFields: e.target.checked })}
+              className={css({ marginTop: '12px' })}>
+              Include non-text fields
+            </Checkbox>
+            <Flex alignItems="end" marginLeft="spacingXs">
+              <Tooltip placement="bottom" content="Includes numbers, dates, and other non-text fields in your search. This may slow down search performance">
+                <HelpCircleIcon />
+              </Tooltip>
+            </Flex>
+          </Flex>
+        </Flex>
+
+        <Flex flexDirection="column">
+          <FormLabel htmlFor="contentTypes">Content Types</FormLabel>
+          <Popover isOpen={contentTypeDropdownOpen} onClose={onContentTypeDropdownClose}>
+            <Popover.Trigger>
+              <Button variant="secondary" ref={anchorRef} className={css({ fontWeight: 'normal' })} onClick={onContentTypeDropdownToggle}>
+                {filters.selectedContentTypes.length === 0 ? 'All Content Types' : `${filters.selectedContentTypes.length} selected`}
+              </Button>
+            </Popover.Trigger>
+
+            <Popover.Content>
+              <List
+                className={css({
+                  listStyle: 'none',
+                  fontWeight: 'normal',
+                  padding: 5,
+                  margin: 0,
+                })}>
+                {contentTypes.map((ct) => {
+                  const isChecked = filters.selectedContentTypes.includes(ct.sys.id);
+                  return (
+                    <ListItem key={ct.sys.id}>
+                      <Checkbox isChecked={isChecked} onChange={() => handleContentTypeToggle(ct.sys.id)}>
+                        {ct.name}
+                      </Checkbox>
+                    </ListItem>
+                  );
+                })}
+              </List>
+            </Popover.Content>
+          </Popover>
+        </Flex>
+
+        <Flex flexDirection="column">
+          <FormLabel htmlFor="locale">Locale</FormLabel>
+          <Select id="locale" value={filters.locale} onChange={(e) => onFiltersChange({ locale: e.target.value })} className={css({ minWidth: '120px' })}>
+            {locales.map((loc) => (
+              <Option key={loc} value={loc}>
+                {loc}
+              </Option>
+            ))}
```

Comment:
this might be a little safer:
```suggestion
            {locales.map((loc, index) => (
              <Option key={index} value={loc}>
                {loc}
              </Option>
            ))}
```

Reply:
I'm not sure about this one. Using index as the key is discouraged unless there is no better identifier. The locales should be both unique and stable, so identifying the options by them should be the safest option.